esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -558,21 +558,25 @@ diagnostics:
   WOA:
     description: WOA check
     variables:
+      sos:
+        mip: Omon
+      tos:
+        mip: Omon
       so:
         mip: Omon
       thetao:
         mip: Omon
-      no3:
-        mip: Oyr
-      o2:
-        mip: Oyr
-      po4:
-        mip: Oyr
-      si:
-        mip: Oyr
+      #no3:
+      #  mip: Oyr
+      #o2:
+      #  mip: Oyr
+      #po4:
+      #  mip: Oyr
+      #si:
+      #  mip: Oyr
```

Comment:
please remove commented-out lines

Reply:
This was jsut for testing. So I removed the commented-out signs again.

Reply:
they're still there, I don't think you pushed the changes - I can remove them if you OK it

Reply:
Okay. Thanks!